test/e2e/kubeTest.sh

```diff
@@ -0,0 +1,162 @@
+# Copyright 2017 Istio Authors
```

Comment:
Sure. As long as you tell @liamawhite how to set it up. I think if its moving to the istio module, then we need to have a different way to get the current manager/mixer image being built/obtain them from docker hub? Then this PR becomes relatively simple, with just prebuilt test images, a shell script that runs through the entire validation setup.